UPGRADE-5.0.md

```diff
@@ -198,3 +198,47 @@ final class CustomRouter implements Router
     }
 }
 ```
+
+## Drivers Should No Longer Call `Envelope::retry`
+
+In 4.X (and lower) drivers were required to call `$envelope->retry()` on any
+envelope passed in `Driver::retry`.
+
+That should now happen in implements of `PMG\Queue\Consumer` instead.
```

Comment:
There isn't an example of this in the documentation. Reading this as if I had never used the package before, I would have no idea how to use retries.

Reply:
Good point, there should be an example for driver authors and otherwise, but really users need not worry about this because it's an internal detail.

Maybe there needs to be an `internals` area in the documentation.